[BUG] Unable to build Morpheus in DEV container

**Describe the bug**
Getting the following error when trying to build Morpheus in DEV container:
```
CMake Error at morpheus/_lib/cmake/libraries/cuda_utils.cmake:15 (add_library):
  Target "cuda_utils" links to target "CUDA::cuda_driver" but the target was
  not found.  Perhaps a find_package() call is missing for an IMPORTED
  target, or an ALIAS target is missing?
Call Stack (most recent call first):
  morpheus/_lib/CMakeLists.txt:36 (include)


CMake Error at morpheus/cmake/utils/python_module_tools.cmake:156 (add_library):
  Target "morpheus._lib.cudf_helpers" links to target "CUDA::cuda_driver" but
  the target was not found.  Perhaps a find_package() call is missing for an
  IMPORTED target, or an ALIAS target is missing?
Call Stack (most recent call first):
  morpheus/cmake/utils/python_module_tools.cmake:260 (add_python_module)
  morpheus/_lib/cmake/libraries/cudf_helpers.cmake:15 (morpheus_add_cython_libraries)
  morpheus/_lib/CMakeLists.txt:39 (include)


CMake Error at morpheus/_lib/cmake/libraries/morpheus.cmake:17 (add_library):
  Target "morpheus" links to target "CUDA::cuda_driver" but the target was
  not found.  Perhaps a find_package() call is missing for an IMPORTED
  target, or an ALIAS target is missing?
Call Stack (most recent call first):
  morpheus/_lib/CMakeLists.txt:42 (include)


CMake Error at /opt/conda/envs/morpheus/share/cmake/pybind11/pybind11Tools.cmake:165 (add_library):
  Target "morpheus._lib.stages" links to target "CUDA::cuda_driver" but the
  target was not found.  Perhaps a find_package() call is missing for an
  IMPORTED target, or an ALIAS target is missing?
Call Stack (most recent call first):
  morpheus/cmake/utils/python_module_tools.cmake:152 (pybind11_add_module)
  morpheus/cmake/utils/python_module_tools.cmake:274 (add_python_module)
  morpheus/_lib/cmake/python_modules/stages.cmake:15 (morpheus_add_pybind11_module)
  morpheus/_lib/CMakeLists.txt:46 (include)


CMake Error at /opt/conda/envs/morpheus/share/cmake/pybind11/pybind11Tools.cmake:165 (add_library):
  Target "morpheus._lib.messages" links to target "CUDA::cuda_driver" but the
  target was not found.  Perhaps a find_package() call is missing for an
  IMPORTED target, or an ALIAS target is missing?
Call Stack (most recent call first):
  morpheus/cmake/utils/python_module_tools.cmake:152 (pybind11_add_module)
  morpheus/cmake/utils/python_module_tools.cmake:274 (add_python_module)
  morpheus/_lib/cmake/python_modules/messages.cmake:15 (morpheus_add_pybind11_module)
  morpheus/_lib/CMakeLists.txt:49 (include)


CMake Error at /opt/conda/envs/morpheus/share/cmake/pybind11/pybind11Tools.cmake:165 (add_library):
  Target "morpheus._lib.common" links to target "CUDA::cuda_driver" but the
  target was not found.  Perhaps a find_package() call is missing for an
  IMPORTED target, or an ALIAS target is missing?
Call Stack (most recent call first):
  morpheus/cmake/utils/python_module_tools.cmake:152 (pybind11_add_module)
  morpheus/cmake/utils/python_module_tools.cmake:274 (add_python_module)
  morpheus/_lib/cmake/python_modules/common.cmake:15 (morpheus_add_pybind11_module)
  morpheus/_lib/CMakeLists.txt:52 (include)


CMake Error at /opt/conda/envs/morpheus/share/cmake/pybind11/pybind11Tools.cmake:165 (add_library):
  Target "morpheus._lib.file_types" links to target "CUDA::cuda_driver" but
  the target was not found.  Perhaps a find_package() call is missing for an
  IMPORTED target, or an ALIAS target is missing?
Call Stack (most recent call first):
  morpheus/cmake/utils/python_module_tools.cmake:152 (pybind11_add_module)
  morpheus/cmake/utils/python_module_tools.cmake:274 (add_python_module)
  morpheus/_lib/cmake/python_modules/file_types.cmake:15 (morpheus_add_pybind11_module)
  morpheus/_lib/CMakeLists.txt:55 (include)


-- Generating done
CMake Generate step failed.  Build files cannot be regenerated correctly.
```

**Steps/Code to reproduce bug**
1. Clone or pull latest from `branch-22.06`
2. Build DEV container: `./docker/build_container_dev.sh`
3. Run DEV container: `./docker/run_container_dev.sh`
4. In container, run `./scripts/compile.sh`

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -62,21 +62,7 @@ rapids_cmake_build_type(Release)
 include(cmake/configure_toolchain.cmake)
 
 # Project definition
-project(morpheus LANGUAGES C CXX)
-
-# Delay enabling CUDA until after we have determined our CXX compiler
-if(NOT DEFINED CMAKE_CUDA_HOST_COMPILER)
-  message(STATUS "Setting CUDA host compiler to match CXX compiler: ${CMAKE_CXX_COMPILER}")
-
-  # Only set the host compiler if we arent using clang. Using clang > 8ish is
-  # incompatible with CUDA 11.4/11.5/11.6. See Issue #102
-  if(NOT "${CMAKE_CXX_COMPILER_ID}" STREQUAL "Clang")
-    set(CMAKE_CUDA_HOST_COMPILER ${CMAKE_CXX_COMPILER})
-  endif()
-endif()
-
-# Now enable CUDA
-enable_language(CUDA)
+project(morpheus LANGUAGES C CXX CUDA)
 
 # Ccache configuration
 include(setup_cache)

--- a/ci/Jenkinsfile
+++ b/ci/Jenkinsfile
@@ -26,7 +26,7 @@ pipeline {
                     when { environment name: 'BUILD_TYPE', value: 'pull-request' }
                     agent {
                         docker {
-                            image 'gpuci/rapidsai-driver:22.06-cuda11.5-devel-centos7-py3.8'
+                            image 'gpuci/rapidsai-driver:22.04-cuda11.5-devel-centos7-py3.8'
                             label 'cpu'
                         }
                     }
@@ -50,7 +50,7 @@ pipeline {
                     }
                     agent {
                         docker {
-                            image 'gpuci/rapidsai:22.06-cuda11.5-devel-centos7-py3.8'
+                            image 'gpuci/rapidsai:22.04-cuda11.5-devel-centos7-py3.8'
                             label 'driver-495'
                             args '--runtime "nvidia" -e "NVIDIA_VISIBLE_DEVICES=$EXECUTOR_NUMBER"'
                         }
@@ -83,7 +83,7 @@ pipeline {
                     }
                     agent {
                         docker {
-                            image 'gpuci/rapidsai:22.06-cuda11.5-devel-centos7-py3.8'
+                            image 'gpuci/rapidsai:22.04-cuda11.5-devel-centos7-py3.8'
                             label 'driver-495'
                             args '--runtime "nvidia" -e "NVIDIA_VISIBLE_DEVICES=$EXECUTOR_NUMBER"'
                         }
@@ -114,7 +114,7 @@ pipeline {
                     }
                     agent {
                         docker {
-                            image 'gpuci/rapidsai:22.06-cuda11.5-devel-centos7-py3.8'
+                            image 'gpuci/rapidsai:22.04-cuda11.5-devel-centos7-py3.8'
                             label 'driver-495'
                             args '--runtime "nvidia" -e "NVIDIA_VISIBLE_DEVICES=$EXECUTOR_NUMBER"'
                         }

